fxa-auth-db-server/index.js

```diff
@@ -107,6 +107,9 @@ function createServer(db) {
   api.get('/securityEvents/:id/ip/:ipAddr', withParams(db.securityEvents))
   api.post('/securityEvents', withBodyAndQuery(db.createSecurityEvent))
 
+  api.get('/emailBounces/:email', op(req => db.fetchEmailBounces(req.params.email)))
```

Comment:
style nit: we try to use `(req) => ` everywhere in the content server

ESLint rule we should probably add: http://eslint.org/docs/rules/arrow-parens